### PR TITLE
Empty Timeline View

### PR DESCRIPTION
**Pull request by sdub18**
An empty view that alerts the user when the app has finished loading and there is no content on that page. For example, when I have no DMs or no Notifications.

This addresses #46

### CODE REVIEW

**Comment by jb55** [comment]
needs rebase :[

**Comment by sdub18** [comment]
@jb55 I thinks its good now? Sorry my rebasing expertise is very little unfortunately

File: damus.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -421,6 +431,7 @@
 		4C75EFA227FA576C0006080F /* Views */ = {
 			isa = PBXGroup;
 			children = (
+				3169CAE4294E699400EE4006 /* Empty Views */,
```
Comment (scoder1747) [inline damus.xcodeproj/project.pbxproj:434]: I see this change here does not have .swift file-extension unlike the rest below.

Comment (sdub18) [reply damus.xcodeproj/project.pbxproj:434]: Yeah this is just a folder. Although good catch cuz I actually wanted to get rid of this.

**Comment by jb55** [comment]
> @jb55 I thinks its good now? Sorry my rebasing expertise is very little unfortunately

ok np I can help with that.

**Comment by sdub18** [comment]
Sounds great. I'd love to zoom or something if you would be willing to show me how you do it? No worries if not though!

**Comment by jb55** [comment]
this is in the latest testflight!

**Comment by scoder1747** [comment]
> this is in the latest testflight!

what is the version # for it?

**Pull request closed without merging**